### ISSUE TITLE
Loading language controlled vocabulary fails

### ISSUE THREAD

**Issue by jcoyne** [samvera/hyrax#1034]
### Descriptive summary

When I attempt to run the controlled vocabulary loading task I get a constraint error:

```
$ rake hyrax:controlled_vocabularies:language
Downloading http://www.lexvo.org/resources/lexvo_2013-02-09.rdf.gz
Extracting /Users/jcoyne/workspace/hyku/tmp/lexvo_2013-02-09.rdf.gz
Importing /Users/jcoyne/workspace/hyku/tmp/lexvo_2013-02-09.rdf
rake aborted!
ActiveRecord::InvalidForeignKey: PG::ForeignKeyViolation: ERROR:  insert or update on table "qa_local_authority_entries" violates foreign key constraint "fk_rails_cee742275b"
DETAIL:  Key (local_authority_id)=(1) is not present in table "local_authorities".
: INSERT INTO "qa_local_authority_entries" ("local_authority_id", "label", "uri", "created_at", "updated_at") VALUES ($1, $2, $3, $4, $5) RETURNING "id"

PG::ForeignKeyViolation: ERROR:  insert or update on table "qa_local_authority_entries" violates foreign key constraint "fk_rails_cee742275b"
DETAIL:  Key (local_authority_id)=(1) is not present in table "local_authorities".

Tasks: TOP => hyrax:controlled_vocabularies:language
```

It looks as though it's expecting a row (`1`) in the local_authorities table, but nothing is there.

**Comment by mjgiarlo** [comment]
Follow up with new ticket to load languages in AWS demo deployment.

**Comment by geekscruff** [comment]
@mjgiarlo @jcoyne ... did you get a solution to this one? I've just hit it.

**Comment by mjgiarlo** [comment]
@geekscruff sadly, no.

**Comment by geekscruff** [comment]
@mjgiarlo I've submitted a [PR ](https://github.com/samvera-labs/hyku/pull/1511) to fix this for Hyku if you have a chance to review. This won't be an issue for Hyrax repos as the foreign keys get setup correctly when generating the repo. AFAIK the problem was only for Hyku so I think this issue can be closed.

**Comment by jcoyne** [comment]
Fixed by #1511
